### DISCUSSION TITLE
Access control

### DISCUSSION THREAD

**Discussion by manojnilanga** [semaphoreui/semaphore#971] [Q&A]
Is there a way to control the access for templates by user based? 

ex: I want to make user1 can access all templates, but user2 can access only 1 template on the ansible semaphore ui.

**Comment by fiftin** [comment]
No, currently it doesn't possible.

**Reply by manojnilanga** [reply]
Is it something, semaphore developing team working on currently? Can we expect it in near future?

**Reply by fiftin** [reply]
Not soon.
